package.json

```diff
@@ -4,9 +4,9 @@
     "lint": "eslint ."
   },
```

Comment:
please, delete all .json files from the commit) and in the future tasks - don't add them to your commits, they are for your local repo)

Reply:
ok